bugbug/model.py

```diff
@@ -18,15 +18,15 @@
 
 
 class Model():
-    def __init__(self, lemmatization=False):
+    def __init__(self, lemmatization=False, ngrams=1):
```

Comment:
Not all models are using ngrams, so this shouldn't be an option of the base model class.